src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```diff
@@ -10,67 +10,36 @@ $schedule-btn--padding: 0.35em 1em;
 
 $schedule-btn--height: 51px;
 
-.NavBarWrapper {
-  position: relative;
-  display: flex;
-  flex-direction: column;
-
-  &__download-buttons {
-    position: fixed;
-    bottom: 0;
-    width: 40%;
-    z-index: z(navbar-schedule-download);
-    display: flex;
-    align-items: center;
-    justify-content: flex-end;
-    background-color: opaque-black(0.9);
-    height: $schedule-btn--height;
-    padding-left: $NavBarSchedule--padding;
-  }
-
-  // @debt: overwrites default btn btn-primary styles
-  & .NavBarWrapper__download-schedule-btn {
-    cursor: pointer;
-    opacity: 0.8;
-
-    margin-right: $spacing--sm;
-
-    text-shadow: 0 2px 4px opaque-black(0.3);
-    box-shadow: 0 3px 6px opaque-black(0.2);
-    padding: $schedule-btn--padding;
-
-    &:hover,
-    &:focus {
-      opacity: 1;
-    }
-  }
-}
-
 .NavBarSchedule {
   position: fixed;
   z-index: z(navbar-schedule);
-  top: 0;
+  top: $navbar-height;
   left: 0;
-  height: calc(100% - #{$schedule-btn--height});
+  bottom: $footer-height;
   width: $NavBarSchedule--width;
 
-  padding-top: $navbar-height;
   padding-left: $NavBarSchedule--padding;
 
   background-color: opaque-black(0.8);
-  overflow: auto;
 
-  pointer-events: none;
+  pointer-events: all;
```

Comment:
This is not being rendered when the Schedule is invisible, so no need for two values.